### ISSUE TITLE
Bot display in chat channels

### ISSUE THREAD

**Issue by FabriceFABS** [springweblobby/swl-website#79]
We don't see bots anymore in any chat channel.
Is it possible to implement an option to see bots by default in channels ?
I use and want to see them in `#[ACE]` (for instance) channel like before.

Thx...

**Comment by db81** [comment]
I'm not sure what you mean by that. Bots don't display in the channel user list? When did that stop working?

**Comment by FabriceFABS** [comment]
Today... from the moment where there's new icons on player's nicks (TS ?) and also from the moment where we can see COUNTRY when mouse is under player's flag.

![bot channel](https://cloud.githubusercontent.com/assets/6024913/18316267/3f2340ae-751a-11e6-9247-12d496ec4d42.jpg)

**Comment by FabriceFABS** [comment]
Humm sorry, I ran another client, and it's the same :/
Sorry, my fault... Anyway I should check what's going on, first time I see that all ACE hosts didn't join / left  "their" channel.

**Comment by FabriceFABS** [comment]
No, it's my fault. In fact, in #[ACE] channel I placed a password with Chanserv.
Today, it has been a SpringServer deco.
All my autohost could not connect back in that channel, because I forgot to fill on them the password for join #[ACE] channel.
It's all ok, nothing regards to SWL, I also removed password I've previously set, finally with Chanserv.

I was just afraid, during an update, you decided not to show bots anymore !
Plz do do it btw, we need it to manage them (join `#[ACE]` if you want to see it)
